frontend/tests/features/collection/revision.test.ts

```diff
@@ -42,7 +42,10 @@ describeIfDeployed("Collection Revision", () => {
       getTestID("revision-tag")
     );
 
-    expect(revisionTag).not.toBe(null);
+    await tryUntil(
```

Comment:
There's a "_global_" timeout in the case these keeps going, right?

Reply:
Yes! Playwright has a global timeout after 30 seconds 🙆‍♂️